docs/reference/ml/trained-models/apis/start-trained-model-deployment.asciidoc

```diff
@@ -58,6 +58,22 @@ include::{es-ref-dir}/ml/ml-shared.asciidoc[tag=model-id]
 [[start-trained-model-deployment-query-params]]
 == {api-query-parms-title}
 
+`adaptive_allocations`::
```

Comment:
I would also refer to `adaptive_allocations` in the description section in the same paragraph as `number_of_allocations`.